reactor-core/src/main/java/reactor/core/publisher/MonoHasElement.java

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
this MonoHasElement change is orthogonal... shelve it and isolate it in another PR / commit on master